lapce-data/src/data.rs

```diff
@@ -228,12 +293,34 @@ impl LapceData {
         }
 
         if let Some((window_id, _)) = windows.iter().next() {
-            for file in files {
-                let _ = event_sink.submit_command(
-                    LAPCE_UI_COMMAND,
-                    LapceUICommand::OpenFile(file.to_path_buf(), false),
-                    Target::Window(*window_id),
-                );
+            for (file, linecol) in dbg!(files) {
```

Comment:
```rust
for (file, linecol) in dbg!(files)
```
You can use dbg!(files) in your local machine, but here it's definitely superfluous.